build.js

```diff
@@ -48,10 +48,11 @@ if (!REPLAY_LOCAL_DRIVER_DIR) {
   let driverArchive = `${currentPlatform()}-recordreplay${archSuffix}.tgz`;
   let downloadArchive = driverArchive;
   let driverRevisionOverride = DRIVER_REVISION;
+  driverRevisionOverride = driverRevisionOverride.substring(0, 12);
```

Comment:
can we remove the duplicate line down on line 58 now?

Reply:
whoops, yes. Good call!

Reply:
Is this guaranteed to never be `undefined`?